proxy/backend/core/src/main/java/org/apache/shardingsphere/proxy/backend/config/ProxyConfigurationLoader.java

```diff
@@ -65,8 +67,13 @@ public static YamlProxyConfiguration load(final String path) throws IOException
         YamlProxyServerConfiguration serverConfig = loadServerConfiguration(getResourceFile(String.join("/", path, SERVER_CONFIG_FILE)));
         File configPath = getResourceFile(path);
         Collection<YamlProxyDatabaseConfiguration> databaseConfigs = loadDatabaseConfigurations(configPath);
+        Collection<YamlHBaseConfiguration> hbaseConfigs = loadHBaseConfigurations(configPath);
         return new YamlProxyConfiguration(serverConfig, databaseConfigs.stream().collect(Collectors.toMap(
-                YamlProxyDatabaseConfiguration::getDatabaseName, each -> each, (oldValue, currentValue) -> oldValue, LinkedHashMap::new)));
+                YamlProxyDatabaseConfiguration::getDatabaseName, each -> each, (oldValue, currentValue) -> oldValue,
+                LinkedHashMap::new)), hbaseConfigs.stream().collect(
+                        Collectors.toMap(
+                                YamlHBaseConfiguration::getDatabaseName, each -> each, (oldValue, currentValue) -> oldValue,
+                                LinkedHashMap::new)));
```

Comment:
HBase and JDBC DataSource should be process in pluggable way. I will refactor here after merged